keadm/cmd/keadm/app/cmd/util/rpminstaller.go

```diff
@@ -51,13 +56,23 @@ func (r *RpmOS) InstallMQTT() error {
 		return nil
 	}
 
-	// install MQTT
-	for _, command := range []string{
-		"yum -y install epel-release",
+	commands := []string{
 		"yum -y install mosquitto",
 		"systemctl start mosquitto",
 		"systemctl enable mosquitto",
-	} {
+	}
+
+	if vendorName, err := getOSVendorName(); err != nil {
+		return err
+	} else {
+		// epel-release package does not included in openEuler
+		if vendorName != openEulerVendorName {
+			commands = append([]string{"yum -y install epel-release"}, commands...)
+		}
+	}
```

Comment:
```suggestion
        commands := []string{
		"yum -y install epel-release",
		"yum -y install mosquitto",
		"systemctl start mosquitto",
		"systemctl enable mosquitto",
	}
	
	vendorName, err := getOSVendorName()
	if err != nil {
		return err
	}
	// epel-release package does not included in openEuler
      if vendorName == openEulerVendorName {
	      commands = commands[1:]
      }
```
I think this coding method will be better.What do you think?

Reply:
OK~